node_pools.tf

```diff
@@ -141,7 +141,7 @@ resource "google_container_node_pool" "node_pool_dynamic_pods" {
   node_config {
 
     labels = {
-      "astronomer.io/multi-tenant" = "true"
+      "astronomer.io/dynamic-pods" = "true"
```

Comment:
should we include both labels? it is multi-tenant after all.

Reply:
No, because we don't want any other pods on this NodePool except the ones created using KubePodOperator & KubernetesExecutor

Reply:
This would allow to scale down the NodePool to 0